manifests/charts/gateways/istio-egress/values.yaml

```diff
@@ -281,3 +281,17 @@ global:
   #   kubernetes: cluster.local
   #   else:  default dns domain
   trustDomain: "cluster.local"
+
+meshConfig:
+  enablePrometheusMerge: true
```

Comment:
This seems to be the default install value - but the current desire is for defaults to be reflected in the code. Can you check what is the default value in code ( and why do we need explicit override ), and maybe reduce this to:

meshConfig:
   defaultConfig: {}